apps/etterlatte-behandling/src/main/kotlin/sak/SakDao.kt

```diff
@@ -64,4 +64,17 @@ class SakDao(private val connection: () -> Connection) {
         statement.setLong(1, id)
         statement.executeUpdate()
     }
+
+    fun markerSakerMedSkjerming(sakIder: List<Long>, skjermet: Boolean) {
+        val statement = connection()
+            .prepareStatement(
+                """
+                UPDATE sak 
+                set erSkjermet = ? 
+                where id in (${sakIder.joinToString(", ") { "'$it'" }}) 
+                """.trimIndent()
+            )
+        statement.setBoolean(1, skjermet)
+        statement.executeUpdate()
```

Comment:
Liker bedre å bruke any på et array for prepared statements, så blir spørringen en faktisk prepared statement og ikke en dynamisk formet sql:

```suggestion
        with(connection()) {
          val statement = prepareStatement(
                  """
                  UPDATE sak 
                  set erSkjermet = ? 
                  where id = any(?) 
                  """.trimIndent()
              )
          statement.setBoolean(1, skjermet)
          statement.setArray(2, createArrayOf("bigserial", sakIder.toTypedArray()))
          statement.executeUpdate()
        }
```